.github/workflows/lint-changelog.yml

```diff
@@ -0,0 +1,25 @@
+name: Lint the changelog
+on: [pull_request]
+
+jobs:
+    lint-changelog:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Cancel Previous Runs
+              uses: styfle/cancel-workflow-action@0.7.0
+              with:
+                  access_token: ${{ github.token }}
+            - name: Check out repository code
+              uses: actions/checkout@v2
+            - name: Check for changelog entry
```

Comment:
The way this works now when running this script, is if there is not a changelog entry it errors so as to report failure on the action but it also exports a label name and action type of remove or add that can be passed along to the github action args. 

the github action always runs regardless of failure thanks to `if: ${{ always() }}`, this means it will always do the labelling but the job will still report failure if the lint check fails.